db/migrate/20160726061957_turn_default_value_to_lookup_value.rb

```diff
@@ -0,0 +1,25 @@
+class TurnDefaultValueToLookupValue < ActiveRecord::Migration
+  def up
+    LookupValue.transaction do
+      LookupKey.unscoped.find_each do |key|
+        LookupValue.create(:match => 'default', :value => key.default_value, :lookup_key_id => key.id, :use_puppet_default => key.use_puppet_default)
+      end
+    end
```

Comment:
@tbrisker, @iNecas thanks.
Using a fake object brought this down to ~11 sec from ~60 sec for 10k lookup keys so I think that is good enough and we can revisit the pure sql if needed.
I haven't tried with the bulk gem but I'll push the fake object change so it can be tested and if needed I'll try out the gem as well.